app/controllers/api/v1/concerns/study_aware.rb

```diff
@@ -7,18 +7,25 @@ module StudyAware
         def set_study
           @study = Study.any_of({accession: params[:study_id]},{id: params[:study_id]}).first
           if @study.nil? || @study.queued_for_deletion?
-            head 404 and return
+            head 404
```

Comment:
I'd be ok keeping the and return. It's superfluous currently, but it's a nice guard to have in case someone in the future adds any logic after the if

Reply:
Rubocop keeps barking at me about it, so I did a little checking.  Calling `head` is analogous to calling `render`, which implicitly ends the action.  Any code after this statement will not execute.  We only need the `and return` when using `redirect_to`, as it [does not halt execution](https://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to).